src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+    private Random rnd = new Random();
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = rnd.nextInt(number);
+    }
```

Comment:
we should generate random number outside of the class Ball
```suggestion
    public Ball(String color, int number) {
        this.color = color;
        this.number = number;
    }
```
```suggestion
    public Ball(String color, int number) {
        this.color = color;
        this.number = rnd.nextInt(number);
    }
```